src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MemberAnalysisResult.cs

```diff
@@ -211,7 +221,7 @@ public static MemberAnalysisResult BadArgumentConversions(ImmutableArray<int> ar
             Debug.Assert(conversions.Length != 0);
             Debug.Assert(badArguments.Length != 0);
             return new MemberAnalysisResult(
-                MemberResolutionKind.BadArguments,
+                MemberResolutionKind.BadArgumentConversion,
```

Comment:
> MemberResolutionKind.BadArgumentConversion, [](start = 15, length = 44)

Why was this change necessary?

Reply:
Never mind. I see why now. 

---
In reply to: [178668382](https://github.com/dotnet/roslyn/pull/25730#discussion_r178668382) [](ancestors = 178668382)